example-runner/src/main/resources/k8s-bash-scripts/runFormattedK8sExample.sh

```diff
@@ -21,7 +21,7 @@ FORMATTER=./formatOutput.sh
 # Run the formatted rest example
 if [ -f $JARFILE ]; then
   if [ -f $FORMATTER ]; then
-    java -Dlogging.level.root=ERROR -Dlogging.level.uk.gov.gchq.palisade.example.runner.runner.RestExample=INFO -Dspring.profiles.active=k8s,rest -jar $JARFILE | $FORMATTER | tee $OUTPUT
+    java -Dlogging.level.root=ERROR -Dlogging.level.uk.gov.gchq.palisade.example.runner.runner.RestExample=INFO -jar $JARFILE | $FORMATTER | tee $OUTPUT
   else
```

Comment:
I would expect the `k8s` profile is still important, indicating that palisade is at `palisade-service` rather than `localhost:8084`, ditto other services